manifests/run.pp

```diff
@@ -385,18 +387,24 @@
 
   if $restart {
     if $ensure == 'absent' {
+      $restart_stop_command = [$docker_command, 'stop', "--time=${stop_wait_time}", $sanitised_title]
+      $restart_stop_onlyif = [$docker_command, 'inspect', $sanitised_title]
```

Comment:
I don't think this will work as expected. `onlyif` will be expecting an array of arrays.

```suggestion
      $restart_stop_onlyif = [[$docker_command, 'inspect', $sanitised_title][
```

Reply:
Did not have any problems defining it as a simple array tbh. And that is pretty weird for a module that throws me 100s of failures for a single misplaced comma. Will try to implement tho.